dags/htan-validate-dag.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+
+import synapseclient
+from airflow.decorators import dag, task
+from airflow.models import Variable
+from airflow.operators.python import get_current_context
+from services.NextflowTowerService import create_and_open_tower_workspace
+from services.AWSService import upload_file_s3
+
+
+@dag(
+    schedule_interval=None,
+    start_date=datetime(2022, 11, 11),
+    catchup=False,
+    default_args={
+        "retries": 2,
+    },
+    tags=["nextflow_tower"],
+)
+def htan_nf_dcqc_dag():
+    @task()
+    def get_synapse_input_file() -> str:
+        """
+        Gets synapse file (input csv) from synapse and saves it in buffer.
+        Passes along dict object with the path to the downloaded file and the name of the file.
+
+        Returns:
+            str: String containing the path to the downloaded file and the name of the file.
+        """
+        # simple param passing from run with config - sets my test file as default for now
+        syn_id = get_current_context()["params"].get("syn_id") or "syn50919899"
```

Comment:
the default synapse file currently is the `input_full.csv` file from the `nf-dcqc` repo, but [located](https://www.synapse.org/#!Synapse:syn50919899) within a test project that I have on Synpase

Reply:
FYI: The second argument to `dict.get()` (assuming that it's indeed a `dict`) is the default value is the key doesn't exist. 

```suggestion
        syn_id = get_current_context()["params"].get("syn_id", "syn50919899")
```

Reply:
added in latest commit